app/scripts/views/NodeViews/ThreeCSG.js

```diff
@@ -223,9 +182,11 @@ define(['backbone', 'underscore', 'jquery', 'BaseNodeView'], function(Backbone,
     // creating this data may be quite slow, we'll need to be careful
     drawChunked: function(geom, list, callback, that){
 
-      var i = 0;
-      var tick = function() {
-        var color, colorLine;
+      var listPossition = 0;
+      var verticesPossition = 0;
+      var facesPossition = 0;
+      var geometry = new THREE.Geometry();
```

Comment:
Typo: Possition is not a word.  `listIndex`, `vertexIndex`, `faceIndex` is probably more accurate, no?